airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/logging/sentry.py

```diff
@@ -0,0 +1,107 @@
+import os
+import sentry_sdk
+import functools
+
+from dagster import OpExecutionContext, get_dagster_logger
+
+
+def setup_dagster_sentry():
+    """
+    Setup the sentry SDK for Dagster if SENTRY_DSN is defined for the environment.
+
+    Additionally TRACES_SAMPLE_RATE can be set 0-1 otherwise will default to 0.
+
+    Manually sets up a bunch of the default integrations and disables logging of dagster
+    to quiet things down.
+    """
+    from sentry_sdk.integrations.argv import ArgvIntegration
+    from sentry_sdk.integrations.atexit import AtexitIntegration
+    from sentry_sdk.integrations.dedupe import DedupeIntegration
+    from sentry_sdk.integrations.logging import LoggingIntegration, ignore_logger
+    from sentry_sdk.integrations.modules import ModulesIntegration
+    from sentry_sdk.integrations.stdlib import StdlibIntegration
+
+    ignore_logger("dagster")
+
+    SENTRY_DSN = os.environ.get("SENTRY_DSN")
+    SENTRY_ENVIRONMENT = os.environ.get("SENTRY_ENVIRONMENT")
+    TRACES_SAMPLE_RATE = float(os.environ.get("SENTRY_TRACES_SAMPLE_RATE", 0))
+
+    if SENTRY_DSN:
+        sentry_sdk.init(
+            dsn=SENTRY_DSN,
+            traces_sample_rate=TRACES_SAMPLE_RATE,
+            environment=SENTRY_ENVIRONMENT,
+            default_integrations=False,
+            integrations=[
+                AtexitIntegration(),
+                DedupeIntegration(),
+                StdlibIntegration(),
+                ModulesIntegration(),
+                ArgvIntegration(),
+                LoggingIntegration(),
+            ],
+        )
+
+
+def log_asset_or_op_context(context: OpExecutionContext):
+    """
+    Capture Dagster OP context for Sentry Error handling
+    """
+    sentry_sdk.add_breadcrumb(
+        category="dagster",
+        message=f"{context.job_name} - {context.op_def.name}",
+        level="info",
+        data={
+            "run_config": context.run_config,
+            "job_name": context.job_name,
+            "op_name": context.op_def.name,
+            "run_id": context.run_id,
+            "retry_number": context.retry_number,
+        },
+    )
```

Comment:
Is this information better served as a context? https://docs.sentry.io/platforms/python/enriching-events/context/

Reply:
I grabbed it from an example, I thought it was useful, however looking at the docs it seems like its more of a debug statement in case you forget to log context 🤔 

Will remove